src/ex4/server/DB/migrations/20220622154325-auto-inc-items-id2.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.removeColumn("Items", "id");
```

Comment:
Bad practice.
Because you're not a production you should add the autoIncrement in the first migration file and just rerun.
If you decided to change a column you should use the changeColumn interface.

Executing `removeColumn` in a real product with already existing data can be catastrophic, it will delete the entire column data.